### PR TITLE
🚮 Cleanup cache-service-worker

### PR DESCRIPTION
**Pull request by alanorozco**
Abandoned long ago, but related artifacts were not removed. See #1199

### CODE REVIEW

File: build-system/server/app.js [hunk 1]
```diff
@@ -1437,21 +1436,18 @@ window.addEventListener('beforeunload', (evt) => {
 /**
  * Serve entry point script url
  */
```
Comment (rsimha) [inline build-system/server/app.js:1438]: This comment is kinda useless, so remove?

**Review by rsimha — APPROVED** [review]
LGTM! Thanks for the cleanup.

Side note: I had filed #24333 a long time ago to track the cleaning up of `app.js`, but it was closed as stale. Might be worth resurrecting if there is scope for more cleanup of this sort.

File: build-system/server/app.js [hunk 2]
```diff
@@ -241,7 +241,6 @@ app.get(
     '/examples/*.(min|max).html',
     '/test/manual/*.(min|max).html',
     '/test/fixtures/e2e/*/*.(min|max).html',
-    '/dist/cache-sw.(min|max).html',
```
Comment (erwinmombay) [inline build-system/server/app.js:244]: going to defer to @jridgewell on final LGTM since i don't know what happened to the cache sw project

**Review by jridgewell — APPROVED** [review]
My poor SW. 😭

**Comment by alanorozco** [comment]
@jridgewell it was a good idea!

**Pull request merged**